src/screens/RDO/AtividadesPendentes/AtividadesPendentes.tsx

```diff
@@ -0,0 +1,62 @@
+import { Container, Content, List, ListItem, Text } from 'native-base'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import HeaderNav from '../../../components/HeaderNav'
+import { ApplicationState } from '../../../store'
+import { Usuario } from '../../../store/ducks/usuarios/types'
+import { ManutencaoRDO } from '../../../store/ducks/rdo/types';
+import { EmpresasState } from '../../../store/ducks/empresas/types';
+
+interface StateProps {
+  empresas: EmpresasState,
+  rdoAtual: ManutencaoRDO,
+}
+
+type Props = StateProps
+
+class AtividadesPendentes extends Component<Props> {
+
+  render() {
+    const { rdoAtual, empresas } = this.props
+    const plantaSelecionaId = rdoAtual.plantaSelecionadaId;
+
+    const empresa = empresas.listaEmpresas.find(empresa => {
+      return empresa.plantas.find(planta => {
+        return planta.id === plantaSelecionaId
+      })
+    })
+
+    const planta = empresa.plantas.find(planta => {
+      return planta.id === plantaSelecionaId
+    })
```

Comment:
Infelizmente esse é mesmo o melhor método pra recuperar essas informações :(
Única coisa que eu acho que poderia melhorar é declarar a planta e atribuir ela uma única vez, como
```suggestion
    let planta;
    const empresa = empresas.listaEmpresas.find(empresa => {
      planta = empresa.plantas.find(planta => {
        return planta.id === plantaSelecionaId
      })
      return planta;
    })
```

Reply:
mas sinceramente precisaria testar o código pra ver se isso funcionaria como o esperado, coisa que eu não fiz aqui. Como é uma melhoria de performance muito pequena, caso vc ache que não valha a pena alterar esse trecho, ta aprovado o review haha